tests/functional/translation/issues/00117.py

```diff
@@ -36,11 +36,11 @@ class F:
     def some_int(self) -> int:
         return 7
 
-class X:
+class G:
```

Comment:
Just to be sure, we don't actually have to rename these any more because of the change in constants.py, right?

Reply:
Yes, no need to change it anymore. However, after adding 'X' to constants.py and verifying that the issue doesn't occur anymore even in the previous version of the test (where 'X' is used as a class name), I felt it would be better to still rename it, following the sequence of class names already in the test file. In order words, jumping to X instead of G shouldn't hint on any particular reason why this class would differ from the others, just because its name doesn't follow the sequence.